### ISSUE TITLE
Privacy Policy document is missing

### ISSUE THREAD

**Issue by d2s** [helsinkijs/helsinkijs.org#10]
As the HelsinkiJS event organisers are collecting event participant data via 3rd party services (such as Meetabit), there should be a clearly written Privacy Policy document from the HelsinkiJS core team itself.

Even while Meetabit has their [Terms of Service](https://meetabit.com/terms) page, it lacks legally required details about transfer of the user data from EU area to other legal zones. [Toughbyte Ltd](https://www.toughbyte.com/) ([2704194-5](https://tietopalvelu.ytj.fi/yritystiedot.aspx?yavain=2578233&tarkiste=B39BCBDBC32E81E85EA69F8AB78F917EDC79FDD6)) operates based from Finland, they have various employees outside of the EU area. That causes potential legal risks for the event organisers, as the event attendee data is transferred to various other countries. Even while the Meetabit website itself does not directly show a full list of the attendees on individual event pages, those details are available via individual user pages. Even if those would not be visible via the website itself, having that data in the database is enough of concern. While the company itself is trustworthy, question is mainly about legal requirements related to GDPR and local data privacy laws of Finland. As of now, the Terms of Service of Meetabit does not fulfil the requirements of GDPR, as it lacks various legally required sections (including mention about data protection officer).

Toughbyte's own [GDPR Privacy Notice](https://www.toughbyte.com/privacy) is clearly written with more care (and says that "All the personal data we have is stored on our databases in the EU"), but as these are two seperate services, they should update Meetabit's Terms to match the current regulations.

**Comment by eemeli** [comment]
I think in this case the issue needs to be raised with Toughbyte, as they're the database controller for Meetabit. HelsinkiJS does not have any registry of personally identifying information, so it would not be appropriate for us to have a privacy policy either -- we can't really make promises on others' behalf.

**Comment by d2s** [comment]
@eemeli Agreed. Mainly have been wondering what has been going on with Meetabit, as almost every other event organiser has stopped using it. Mainly question is what is the future of the service, if there haven't really been any visible development on it since many years ago.
